packages/gatsby/cache-dir/fast-refresh-overlay/components/build-error.js

```diff
@@ -6,12 +6,14 @@ import { prettifyStack, openInEditor } from "../utils"
 // Error that is thrown on e.g. webpack errors and thus can't be dismissed and must be fixed
 export function BuildError({ error }) {
   // Incoming build error shape is like this:
+  // Sometimes "Enter"
   // ./relative-path-to-file
   // Additional information (sometimes empty line => handled in "prettifyStack" function)
   // /absolute-path-to-file
   // Errors/Warnings
-  const [file, ...rest] = error.split(`\n`)
-  const decoded = prettifyStack(rest)
+  const decoded = prettifyStack(error)
```

Comment:
The ESLint errors have another shape than other compile errors. But they both have in common that `decoded[0].content` contains the filePath.